### PR TITLE
chore: Update DL notebook

### PR DESCRIPTION
**Pull request by ilan-gold**
That was painful! But the accuracy is better than the last notebook so I think it went alright!

### CODE REVIEW

File: annbatch.ipynb [hunk 1]
```diff
@@ -0,0 +1,1978 @@
+{
```
Comment (Zethson) [inline annbatch.ipynb:1]: <p>pytorch -&gt; PyTorch - also in the title.</p><p> </p><p>I generally miss a 1-2 sentence intro on what this tutorial is about. This starts with bulletpoints that are a tiny bit random. It also starts very technical with "in-memory sparse data" which makes sense but people that are trying to train their first model of AnnData objects will be like: ???</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601820'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>You can use cell tags to hide this stuff or make it optional. It's not the focus of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601824'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # The code is based on the scarches trvae model
</pre><p>That's a lot of lines for not a lot of content in this section (the comments)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601826'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    class CVAE(nn.Module):
</pre><p>See above. This isn't relevant here and could also be a single line import. Maybe hide with cell tags.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601828'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    covid = sc.read('covid_cite.h5ad')
</pre><p>Put this onto our S3 (all of our datasets should be there) and use the backup URL thingy of <code>read</code>,please.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601831'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>You've shown the datasets just above and now you're showing them again. This is unnecessary after just HVG. It's clutter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601835'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>There's a few code cells here but not really any motivation or explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601842'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='21'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>could use sphinx refs for the loader class here.</p><p> </p><p>umaps or UMAPs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601846'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='28'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>A few magic numbers. When it comes to DL these are always questioned so they should be justified</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601850'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='29'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    
</pre><p>Remove the empty lines here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601854'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='33'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.        for batch in train_loader:
</pre><p>This whole section is very long. Can we reduce it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601860'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>Use <code>hide-output</code>,please.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601868'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (Zethson) [inline annbatch.ipynb:1]: <p>Move this into the cell just above to have a little less things rendered and easier scrolling</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/26/discussion/#comment-3404601877'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='42'/>

Comment (ilan-gold) [reply annbatch.ipynb:1]: <p>These are justified in the above. Most people use 256 and I do say "This is a larger batch size..."</p>

Comment (ilan-gold) [reply annbatch.ipynb:1]: <p>I think it is important because it shows how to work with the loader output. Isn't that the point?</p><p> </p>

Comment (ilan-gold) [reply annbatch.ipynb:1]: <p>I think it's pretty common to print this as a sanity check.</p><p> </p>

Comment (Zethson) [reply annbatch.ipynb:1]: <p>I only see the chunk size but not the batch size or preload chunks. Maybe I'm blind. Sorry! It's kinda explained but maybe I was just missing the explicit numbers. Consider it solved then, please.</p>

Comment (ilan-gold) [reply annbatch.ipynb:1]: Sorry, what is a "URL thingy of read"?

Comment (ilan-gold) [reply annbatch.ipynb:1]: I added a bit more!

Comment (ilan-gold) [reply annbatch.ipynb:1]: Ah I see. No I just went with a `pooch` downloader, sorry I assumed this was pointing at `gdown`.

**Comment by Zethson** [comment]
I'll have a look and potentially make some minor edits. Then let's get it in.

**Comment by Zethson** [comment]
@ilan-gold good from my side!
